Solutions/LastPass/Data Connectors/LastPassAPIConnector.json

```diff
@@ -1,123 +1,132 @@
-                        },
-                        "response": {
-                            "eventsJsonPaths": [
-                                "$.data"
-                            ],
-                            "successStatusJsonPath": "$.status",
-                            "successStatusValue": "OK",
-                            "convertChildPropertiesToArray": true
-                        },
-                        "paging": {
-                            "pagingType": "NextPageToken",
-                            "nextPageParaName": "next",
-                            "nextPageTokenJsonPath": "$.next"
-                        }
-                    }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspace": {
+      "type": "string",
+      "defaultValue": ""
+    }
+  },
+  "resources": [
+    {
+      "id": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.OperationalInsights/workspaces/',parameters('workspace'),'/providers/Microsoft.SecurityInsights/dataConnectors/',guid(subscription().subscriptionId))]",
+      "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',guid(subscription().subscriptionId))]",
+      "apiVersion": "2021-03-01-preview",
+      "type": "Microsoft.OperationalInsights/workspaces/providers/dataConnectors",
+      "kind": "APIPolling",
+      "properties": {
+        "connectorUiConfig": {
+          "title": "LastPass Enterprise - Reporting (test)",
+          "publisher": "The Collective Consulting BV",
+          "descriptionMarkdown": "The [LastPass Enterprise](https://www.lastpass.com/products/enterprise-password-management-and-sso) connector provides the capability to LastPass reporting (audit) logs into Azure Sentinel. The connector provides visibility into logins and activity within LastPass (such as reading and removing passwords).",
+          "graphQueriesTableName": "LastPass_BYOC_CL",
+          "graphQueries": [
+            {
+              "metricName": "Total data received",
+              "legend": "LastPass Audit Events",
+              "baseQuery": "{{graphQueriesTableName}}"
+            }
+          ],
+          "sampleQueries": [
+            {
+              "description": "Password moved to shared folders",
+              "query": "{{graphQueriesTableName}}\n | where Action_s == \"Move to Shared Folder\"\n | extend AccountCustomEntity = Username_s, IPCustomEntity = IP_Address_s, URLCustomEntity = Data_s, TimestampCustomEntity = todatetime(Time_s) "
+            }
+          ],
+          "dataTypes": [
+            {
+              "name": "{{graphQueriesTableName}}",
+              "lastDataReceivedQuery": "{{graphQueriesTableName}}\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
             }
+          ],
+          "connectivityCriterias": [
```

Comment:
@ThijsLecomte - Please change the property "connectivityCriterias" to "connectivityCriteria". There shouldn't be additional 's' as the end for this property